agent/engine/execcmd/manager_init_task_linux.go

```diff
@@ -211,6 +243,35 @@ func getSessionWorkersLimit(ma apicontainer.ManagedAgent) int {
 	return limit
 }
 
+var GetExecAgentLogConfigFile = getAgentLogConfigFile
```

Comment:
does this need to be exported?

Reply:
yes, used in engine integ test

Reply:
This is probably for another PR, but I don't like these package level functions when they're exported because any other package could override it and break the functionality of this package...

Maybe instead of using this function in integ test, we could just use a regex. I think the main test that needs to check the actual hash is the UT. The integ test can be more relaxed about the specifics since typically inte tests are gray, if not black box tests